defaults/main.yml

```diff
@@ -36,6 +36,12 @@ stackdriver_apache_servers:
 stackdriver_elasticsearch_enabled: no
 stackdriver_elasticsearch_url: http://localhost:9200/_nodes/_local/stats/
 
+### jvm
+
+stackdriver_jvm_enabled: no
+stackdriver_jvm_host: localhost
+stackdriver_jvm_port: 5000
```

Comment:
Every other service (memcached, MongoDB, PostgreSQL, Redis etc.) all have a standard default port which is industry convention (and those are the ports which are specified here). In the absence of such a industry wide default, we'd want this to be delegated expressly to the customer as something they need to initialize in their playbook.